### PR TITLE
Fix cross version tests related to r86/ProblemProgressEventCrossVersi…

### PR DESCRIPTION
**Pull request by reinsch82**
Fixes 
 - gradle/gradle-private#4182

### Reviewing cheatsheet

Before merging the PR, comments starting with 
- ❌ ❓**must** be fixed
- 🤔 💅 **should** be fixed
- 💭 **may** be fixed
- 🎉 celebrate happy things

### CODE REVIEW

**Comment by reinsch82** [comment]
@bot-gradle test ACT

File: platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r86/ProblemProgressEventCrossVersionTest.groovy [hunk 1]
```diff
@@ -68,30 +63,12 @@ class ProblemProgressEventCrossVersionTest extends ToolingApiSpecification {
...
 
-    static def assertProblemDetailsForTAPIProblemEvent(List<?> problems, String expectedDetails = null, String expectedDocumentation = null) {
-        assertProblemDetailsForTAPIProblemEventWithoutSolution(problems, expectedDetails, expectedDocumentation)
-        problems[0].solutions[0].solution == 'try this instead'
+    static def assertProblemDetailsForTAPIProblemEvent(List<?> problems) {
+        assertProblemDetailsForTAPIProblemEventWithoutSolution(problems)
     }
 
-    static void assertProblemDetailsForTAPIProblemEventWithoutSolution(List<?> problems, String expectedDetails, String expectedDocumentation) {
+    static void assertProblemDetailsForTAPIProblemEventWithoutSolution(List<?> problems) {
```
Comment (donat) [inline platforms/ide/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r86/ProblemProgressEventCrossVersionTest.groovy:70]: Does it bring any value to keep this method? If it's trivial, I'd just inline the `problems.size() == 1` assertion.

**Review by donat — CHANGES_REQUESTED** [review]

**Comment by reinsch82** [comment]
@bot-gradle test ACT

**Comment by reinsch82** [comment]
@bot-gradle test ACT

**Review by donat — APPROVED** [review]

**Pull request merged**